### ISSUE TITLE
position of pause/play button should be placed first

### ISSUE THREAD

**Issue by damiengomez** [Orange-OpenSource/Orange-Boosted-Bootstrap#1578]
### Prerequisites

- [X] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

On the exemple of an accessible carousel with pause/play button, the button is placed behind carousel.
The good pratice of accessibility say that this button should be placed first to let users choose to pause or play it before the carousel.

### Motivation and context

Users of assistives technologies need to have the choice to pause or play a carousel before receiving the information.

**Comment by julien-deramond** [comment]
@Aniort thoughts?

**Comment by Aniort** [comment]
Nice, excellent remark, we should raise an issue to correct ! @julien-deramond

**Comment by julien-deramond** [comment]
Let's tackle it when we will try to introduce the play/pause button within the carousel.

**Comment by julien-deramond** [comment]
After a discussion with designers, developers and a11y experts, we're going to integrate the play/pause button within the carousel before the carousel indicators. The target for this issue will be the v5.3.
